UserManual.md

```diff
@@ -0,0 +1,49 @@
+# QueryForge
+**User Manual**
+
+## Introduction
+QueryForge web application helps students and researchers find suitable research projects and faculty mentors within the ECE department. By gathering information from university websites and lab pages, the tool provides a one-stop platform for exploring research opportunities based on user-defined interests. The app also offers a resume parser that matches user skills and keywords with relevant projects.
+
+---
+
+## Accessing the Application
+1. **Open the Web Application**: Go to the application URL (http://3.142.98.164:5000/) in your preferred web browser.
+2. **Login**: Sign in using your credentials. If you are new to the the application, you will be asked to create an account.
+---
+
+## Features and Usage
+
+### Home Page
+- After logging in, the **Home Page** displays a search bar, filters, and current highlights of research opportunities within the university.
+- Users can search directly for professors, projects, or keywords to see available opportunities.
+
+### Searching for Research Opportunities
+1. **Enter Search Terms**: Type keywords or research areas into the search bar to find related projects or professors.
+
+### Results
+- Use tabs to narrow search results based on:
+  - **Professor**: Shows projects supervised by specific professors.
+  - **Project Type**: VIP/EPICS or professor-mentored projects.
+
+After using the tabs, click **Search** to see refined results.
+
+### Uploading a Resume for Parsing
+1. **Navigate to the Resume Parsing Section**: Access this through the main menu.
+2. **Upload Resume**: Choose a `.pdf` or `.docx` file from your device.
+3. **Parse and Analyze**: Once uploaded, the application will extract text from the resume, highlight key skills, and display the most frequently mentioned keywords.
+
+> **Note**: For optimal parsing, ensure your resume is formatted simply, with clear sections such as **Experience**, **Education**, and **Skills**.
+
+### Viewing Matches
+- **Project Matches**: After parsing, the application will display research projects that align with the keywords in your resume.
+
```

Comment:
Can you add details about the login and sign up page: Like one sentence is also enough I think ? Also I think adding screenshots of annoted webpages for each of our wepgae would be nice.

Reply:
Good suggestion. Added!